src/main/java/salesforce/ui/pages/abstracts/campaign/NewCampaignAbstract.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * @(#) NewCampaignAbstract.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ *  Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ *
+ */
+
+package salesforce.ui.pages.abstracts.campaign;
+
+import salesforce.entities.Campaign;
+import salesforce.ui.BasePage;
+import salesforce.ui.PageFactory;
+import salesforce.utils.StrategySetter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * NewCampaignAbstract.
+ * @author Regis Humana
+ * @version 0.0.1
+ */
+public abstract class NewCampaignAbstract extends BasePage {
+
+    private final String DESCRIPTION = "Description";
```

Comment:
You should have a class with this constants, you are also using them in the entity